lib.go

```diff
@@ -10,11 +11,11 @@ ConfigureEasy - default pipeline setup for configure your structure
 func ConfigureEasy(
 	structure interface{},
 	fileName string) (interface{}, error) {
-	return pipeline.Configure(structure, fileName, []pipeline.FuncType{
-		pipeline.FunctionSetupEnvironment,
-		pipeline.FunctionSetupHocon,
-		pipeline.FunctionSetupDefault,
-	}, pipeline.EmptyAdditionalPrefix)
+	return pipeline.Configure(structure, fileName, []infra.FuncType{
+		infra.FunctionSetupEnvironment,
+		infra.FunctionSetupHocon,
+		infra.FunctionSetupDefault,
+	}, pipeline.EmptyAdditionalPrefix, pipeline.DurtyConfiguring)
```

Comment:
It's better to use RawConfiguration, or at least spell it dirty

Reply:
++